backend/main.py

```
@@ -1,66 +1,176 @@
-        break
-
-#Run the monitor_resume_signal function in a separate thread
-threading.Thread(target=monitor_resume_signal, daemon=True).start()
+import cv2
+import math
+import asyncio
+from fastapi import FastAPI, WebSocket
+from fastapi.middleware.cors import CORSMiddleware
+import uvicorn
+import base64
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)